docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Health.adoc

```diff
@@ -19,18 +19,45 @@ element to the xml or by using the following CLI operation:
 [standalone@localhost:9990 /]/extension=org.wildfly.extension.microprofile.health-smallrye:add
 ----
 
-== Management Operation
+== Management Operations
+
+The healthiness of the application server can be queried by calling 3 different operations:
+
+* `check` to check both the liveness and readiness of the runtime
+* `check-live` to check only the liveness of the runtime
+* `check-ready` to check only the readiness of the runtime
 
-The healthiness of the application server can be queried by calling the `check`:
 
 [source,options="nowrap"]
 ----
 [standalone@localhost:9990 /] /subsystem=microprofile-health-smallrye:check
 {
-    "outcome" => "success", <1>
+    "outcome" => "success",
     "result" => {
-        "outcome" => "UP", <2>
-        "checks" => []
+        "status" => "UP",
+        "checks" => [
+            {
+                "name" => "empty-liveness-checks",
+                "status" => "UP"
+            },
+            {
+                "name" => "server-state",
+                "status" => "UP",
+                "data" => {"value" => "running"}
+            },
+            {
+                "name" => "boot-errors",
+                "status" => "UP"
+            },
+            {
+                "name" => "deployments-status",
+                "status" => "UP"
+            },
+            {
+                "name" => "empty-readiness-checks",
```

Comment:
So why does this show up in WF21. In previous version it did not show?

Reply:
There was a bug in the code that I fixed in this PR

Reply:
Thanks that explain some changes in tests, which I wasn't expecting.